src/components/BulkUploadListItem.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { CheckCircleIcon, XCircleIcon } from '@heroicons/react/24/outline';
+import { BulkUpload } from '../interfaces/BulkUpload';
+import './BulkUploadListItem.css';
+import { Spinner } from './Spinner';
+
+interface BulkUploadListItemProps {
+  upload: BulkUpload;
+}
+
+export const BulkUploadListItem = ({ upload }: BulkUploadListItemProps) => {
+  const {
+    fileName, fileSize, status, createdAt,
+  } = upload;
+
+  // TODO: Normalize these against the statuses returned by the API.
```

Comment:
Here's the full list from https://github.com/PhilanthropyDataCommons/service/blob/eeb035f4a71cedf7fdeb5226b770349df853272a/src/types/BulkUpload.ts#L5-L9 :

- `pending`
- `in_progress`
- `completed`
- `failed`
- `canceled`

The `canceled` status is planned but not implemented, and the `pending` and `in_progress` can probably both be considered `processing` for the sake of the UI.

Reply:
Oh excellent, I almost guessed correctly! Thanks for surfacing those.

1. Is `canceled` planned for MVP? If so, I'll add styling for it. What does it mean? cc @slifty 
2. You're right that from our end user perspective, `pending` and `in_progress` are the same. (At least, going on my assumption about what they mean. Pending is received and in the queue, but not yet started; in progress has actually been started?)

Reply:
I'm gonna update based on this list and push again.

Reply:
Just confirmation that we can skip canceled (in future the front end could add a "cancel" button for items that are pending, if you wanted.)

Reply:
> Pending is received and in the queue, but not yet started; in progress has actually been started

Correct!